vcsclient/gitlab.go

```diff
@@ -340,32 +339,48 @@ func (client *GitLabClient) DeletePullRequestComment(ctx context.Context, owner,
 
 // GetLatestCommit on GitLab
 func (client *GitLabClient) GetLatestCommit(ctx context.Context, owner, repository, branch string) (CommitInfo, error) {
+	commits, err := client.GetCommits(ctx, owner, repository, branch)
```

Comment:
same code on all providers, can we unify?

Reply:
@sverdlov93, each call goes to a specific implementation in go (go-github, go-gitlab, etc.)